src/plugins/intel_cpu/src/nodes/scatter_update.cpp

```diff
@@ -823,24 +922,41 @@ void ScatterUpdate::scatterUpdate(uint8_t *indices, uint8_t *update, int axis, u
     });
 }
 
+void ScatterUpdate::scatterNDUpdate(const MemoryPtr& dstMemPtr, const MemoryPtr& indicesMemPtr, const MemoryPtr& updateMemPtr) {
+    using namespace scatter_nd_update;
+    ScatterNDUpdateContext ctx{this, dstMemPtr, indicesMemPtr, updateMemPtr, reduction_type};
+    OV_SWITCH(intel_cpu,
```

Comment:
Need to check precision in the fn `initSupportedPrimitiveDescriptors` on line #210